docs/postman/Smart Village API.postman_collection.json

```diff
@@ -1,10 +1,461 @@
+											"type": "string"
+										}
+									]
+								},
+								"method": "PUT",
+								"header": [],
+								"body": {
+									"mode": "raw",
+									"raw": "{\r\n\"category\" :{\r\n    \"title\":\"Plantt\",\r\n    \"description\": \"This is fourth category of welcome space\",\r\n    \"sort_order\" : 1\r\n  }\r\n}",
+									"options": {
+										"raw": {
+											"language": "json"
+										}
+									}
+								},
+								"url": {
+									"raw": "{{API_URL}}/api/v2/category/7?cguid=30fce8a1-99d6-4147-846d-4067409aa9c6",
+									"host": [
+										"{{API_URL}}"
+									],
+									"path": [
+										"api",
+										"v2",
+										"category",
+										"7"
+									],
+									"query": [
+										{
+											"key": "cguid",
+											"value": "30fce8a1-99d6-4147-846d-4067409aa9c6"
+										}
+									]
+								}
+							},
+							"response": []
+						},
+						{
+							"name": "Delete category by id",
+							"request": {
+								"auth": {
+									"type": "bearer",
+									"bearer": [
+										{
+											"key": "token",
+											"value": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJpYXQiOjE2NTMzODcwNjEsImlzcyI6Imh0dHA6XC9cL2xvY2FsaG9zdFwvaHVtaHViIiwibmJmIjoxNjUzMzg3MDYxLCJ1aWQiOjEsImVtYWlsIjoiYW5raXQ0Z2F1cmF2QGdtYWlsLmNvbSJ9.yM3x33-7FenzKGuZFrE5-LPftcfvHCP04KrhaKjXVl1K8ejkZ8K4Benj4QZDPRTI735K-65RasFbhLnm0nXMDw",
+											"type": "string"
+										}
+									]
+								},
+								"method": "DELETE",
+								"header": [],
+								"url": {
+									"raw": "{{API_URL}}/api/v2/delete/category/7?cguid=30fce8a1-99d6-4147-846d-4067409aa9c6",
+									"host": [
+										"{{API_URL}}"
+									],
+									"path": [
+										"api",
+										"v2",
+										"delete",
+										"category",
+										"7"
+									],
+									"query": [
+										{
+											"key": "cguid",
+											"value": "30fce8a1-99d6-4147-846d-4067409aa9c6"
+										}
+									]
+								}
+							},
+							"response": []
+						}
+					]
+				},
+				{
+					"name": "Link",
+					"item": [
+						{
+							"name": "List all links",
+							"request": {
+								"auth": {
+									"type": "bearer",
+									"bearer": [
+										{
+											"key": "token",
+											"value": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJpYXQiOjE2NTMzODcwNjEsImlzcyI6Imh0dHA6XC9cL2xvY2FsaG9zdFwvaHVtaHViIiwibmJmIjoxNjUzMzg3MDYxLCJ1aWQiOjEsImVtYWlsIjoiYW5raXQ0Z2F1cmF2QGdtYWlsLmNvbSJ9.yM3x33-7FenzKGuZFrE5-LPftcfvHCP04KrhaKjXVl1K8ejkZ8K4Benj4QZDPRTI735K-65RasFbhLnm0nXMDw",
+											"type": "string"
+										}
+									]
+								},
+								"method": "GET",
+								"header": [],
+								"url": {
+									"raw": "{{API_URL}}/api/v2/links?cguid=30fce8a1-99d6-4147-846d-4067409aa9c6",
```

Comment:
is the `cguid` needed in any of the links endpoints? i cannot find usage of `cguid` in the `LinkController`. what do i miss?

Reply:
Hi @donni106 
LinkController extends the BaseContentContainerController file where we are defined user token validation code, pagination, success and error response code etc.

And this  BaseContentContainerController file (we created) extends the ContentContainerController file (Humhub core file) in which this cguid is defined.

Note : we cannot extends two file in a single file in php. and same logic is also implemented in categoryController

Permission for fetching, edit, create and delete operation in links and categories are internally handle by humhub itself. we don't need to add addtional logic.